src/command_modules/azure-cli-botservice/azure/cli/command_modules/botservice/_help.py

```diff
@@ -0,0 +1,206 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from knack.help_files import helps
+
+helps['bot'] = """
+    type: group
+    short-summary: Manage Microsoft Bot Services.
+"""
+helps['bot create'] = """
+    type: command
+    short-summary: Create a new Microsoft Bot Service.
```

Comment:
Outside of toplevel descriptions we tend to avoid branding language in CLI help. Should be reverted to `bot`.